threewisemonkeys-as_robo_interact_s2_159.py

```
@@ -0,0 +1,117 @@
+def health_check():
+    return jsonify({'status': 'healthy', 'model_loaded': mask_generator is not None})
+
+if __name__ == '__main__':
+    # Initialize the mask generator once when the server starts
+    sam_checkpoint = sys.argv[1]
+    logger.info("Loading SAM model...")
+    mask_generator = get_sam_mask_generator("sam_vit_b", sam_checkpoint)
+    logger.info("Server ready to accept requests")
+    
+    # Start the Flask server
+    app.run(host='0.0.0.0', port=5000, debug=False)
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)